### PR TITLE
Use tracing for profiling

### PR DESCRIPTION
**Pull request by Bastacyclop**

### CODE REVIEW

File: 10-benchmark/slotted-rise/Cargo.toml [hunk 1]
```diff
@@ -3,10 +3,20 @@ name = "slotted-rise"
...
 edition = "2021"
 
+[features]
+tracing = ["slotted-egraphs/tracing"]
+# ??? no_tracing = ["slotted-egraphs/no_tracing"]
+explanations = ["slotted-egraphs/explanations"]
+
 [dependencies]
-slotted-egraphs = "=0.0.15"
+slotted-egraphs = { path = "../../../slotted-egraphs" }
```
Comment (memoryleak47) [inline 10-benchmark/slotted-rise/Cargo.toml]: Do we want this?
I mean I also do this occasionally. But maybe not the pushed version. It makes it harder to reproduce results long-term.

Comment (Bastacyclop) [reply 10-benchmark/slotted-rise/Cargo.toml]: Happy to bump to the version that includes https://github.com/memoryleak47/slotted-egraphs/pull/3

Comment (memoryleak47) [reply 10-benchmark/slotted-rise/Cargo.toml]: right, I'll quickly do it!

Comment (memoryleak47) [reply 10-benchmark/slotted-rise/Cargo.toml]: it's up

Comment (memoryleak47) [reply 10-benchmark/slotted-rise/Cargo.toml]: v0.0.18: but the feature is called "trace" to not collide with the crate "tracing"

Comment (Bastacyclop) [reply 10-benchmark/slotted-rise/Cargo.toml]: done

**Pull request merged**